app/src/main/res/layout/activity_add_widget_instructions.xml

```diff
@@ -20,177 +22,36 @@
-                android:layout_width="18dp"
-                android:layout_height="18dp"
-                android:layout_margin="16dp"
-                android:background="@drawable/background_add_widget_number"
-                android:fontFamily="sans-serif-black"
-                android:gravity="center"
-                android:text="@string/addWidgetInstruction3Label"
-                android:textColor="@color/white"
-                android:textSize="10sp"
-                android:textStyle="normal"
-                app:layout_constraintBaseline_toBaselineOf="@id/instruction3"
-                app:layout_constraintEnd_toStartOf="@id/instruction3"
-                tools:ignore="SmallSp" />
-
-            <TextView
-                android:id="@+id/instruction3"
-                android:layout_width="0dp"
-                android:layout_height="wrap_content"
-                android:fontFamily="sans-serif"
-                android:lineSpacingExtra="4sp"
-                android:paddingBottom="42dp"
-                android:text="@string/addWidgetInstruction3"
-                android:textColor="@color/white"
-                android:textSize="14sp"
-                android:textStyle="normal"
-                app:layout_constraintEnd_toEndOf="@id/instructionsEndBarrier"
-                app:layout_constraintStart_toStartOf="@id/instructionsStartBarrier"
-                app:layout_constraintTop_toBottomOf="@+id/instruction2"
-                app:layout_constraintWidth_max="400dp" />
-
-            <View
-                android:layout_width="match_parent"
-                android:layout_height="1dp"
-                android:background="@color/divider"
-                app:layout_constraintBottom_toTopOf="@id/homeButton"
-                app:layout_constraintTop_toBottomOf="@id/instruction3" />
-
-            <Button
-                android:id="@+id/closeButton"
-                style="?android:attr/borderlessButtonStyle"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:text="@string/addWidgetInstructionsButtonClose"
-                android:textColor="@color/white"
-                android:textSize="13sp"
-                app:layout_constraintEnd_toStartOf="@id/homeButton"
-                app:layout_constraintTop_toBottomOf="@id/instruction3" />
-
-            <Button
-                android:id="@+id/homeButton"
-                style="?android:attr/borderlessButtonStyle"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:text="@string/addWidgetInstructionsButtonGoHome"
-                android:textColor="@color/white"
-                android:textSize="13sp"
-                app:layout_constraintEnd_toEndOf="parent"
-                app:layout_constraintTop_toBottomOf="@id/instruction3" />
+            android:layout_centerInParent="true"
```

Comment:
This is flagged as invalid layout param

Reply:
Ahh thanks, that's not needed! I've removed it.